pkg/kubelet/pleg/generic.go

```diff
@@ -186,9 +212,12 @@ func (g *GenericPLEG) updateRelistTime(timestamp time.Time) {
 	g.relistTime.Store(timestamp)
 }
 
-// relist queries the container runtime for list of pods/containers, compare
+// Relist queries the container runtime for list of pods/containers, compare
 // with the internal pods/containers, and generates events accordingly.
-func (g *GenericPLEG) relist() {
+func (g *GenericPLEG) Relist() {
+	g.relistLock.Lock()
```

Comment:
Sorry if this was already mentioned, but why is this mutex necessery? What fields is it guarding? Isn't this function only called by the go-routine spawned in the wait.Until in `Start()`?

Reply:
agree with @bobbypage it would be important to clarify this before beta.